metrictank/src/main/java/com/expedia/metrics/metrictank/MetricTankKafkaDeserializer.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.metrics.metrictank;
+
+import com.expedia.metrics.MetricData;
+import com.expedia.metrics.MetricDefinition;
+import com.expedia.metrics.TagCollection;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.expedia.metrics.MetricDefinition.MTYPE;
+import static com.expedia.metrics.MetricDefinition.UNIT;
+
+public class MetricTankKafkaDeserializer extends AbstractMetricTankDeserializer<MetricData> {
```

Comment:
A MetricPoint does not have enough information to be converted directly back into a MetricData. You can delete this whole class. Bonus: Then you can combine AbstractMetricTankDeserializer and MetricPointKafkaDeserializer into a single class.

We will need a way to turn MetricPoint into MetricData, but I'll address that with a caching serialiser in a separate PR